Update fence_gates/wooden

nitrodynamite18 opened the pull request:
Added "minecraft:mangrove_fence_gate" to fences/wooden.json

nitrodynamite18 committed:
Update wooden.json

Added "minecraft:mangrove_fence_gate"

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/MinecraftForge/MinecraftForge?pullRequest=8936) <br/>All committers have signed the CLA.

AterAnimAvis reviewed (changes requested):
👋 Hello, thank you for your contribution, that file is actually generated via [datagen](https://github.com/MinecraftForge/MinecraftForge/blob/0e66b7b21757e510aef706ca320881b9eb268822/src/main/java/net/minecraftforge/common/data/ForgeBlockTagsProvider.java#L49), modifying and running datagen instead you'd also fix the matching `items/fences/wooden` which your missing here.

nitrodynamite18 committed:
Update datagen

nitrodynamite18 committed:
Update datagen

nitrodynamite18 commented:
As requested, updated datagen and copied the fix to the other relevant tag.

Sirttas commented on src/main/java/net/minecraftforge/common/data/ForgeBlockTagsProvider.java:
```diff
@@ -46,7 +46,7 @@ public void addTags()
         tag(ENDERMAN_PLACE_ON_BLACKLIST);
         tag(FENCE_GATES).addTags(FENCE_GATES_WOODEN);
-        tag(FENCE_GATES_WOODEN).add(Blocks.OAK_FENCE_GATE, Blocks.SPRUCE_FENCE_GATE, Blocks.BIRCH_FENCE_GATE, Blocks.JUNGLE_FENCE_GATE, Blocks.ACACIA_FENCE_GATE, Blocks.DARK_OAK_FENCE_GATE, Blocks.CRIMSON_FENCE_GATE, Blocks.WARPED_FENCE_GATE);
+        tag(FENCE_GATES_WOODEN).addTags(Blocks.OAK_FENCE_GATE, Blocks.SPRUCE_FENCE_GATE, Blocks.BIRCH_FENCE_GATE, Blocks.JUNGLE_FENCE_GATE, Blocks.ACACIA_FENCE_GATE, Blocks.DARK_OAK_FENCE_GATE, Blocks.CRIMSON_FENCE_GATE, Blocks.WARPED_FENCE_GATE, Blocks.MANGROVE_FENCE_GATE);
```
Have you tried running datagen? Because I'm 99% sure  changeing `add` to `addTags` wont compile at all.

nitrodynamite18 replied:
Fixed.

nitrodynamite18 committed:
Update ForgeBlockTagsProvider.java

sciwhiz12 commented on src/generated/resources/data/forge/tags/blocks/fence_gates/wooden.json:
```diff
@@ -7,6 +7,7 @@
   ]
-}
\ No newline at end of file
+}
```
Please rerun datagen to remove this added end of file newline.

nitrodynamite18 committed:
EOF removed.

nitrodynamite18 committed:
Update wooden.json

nitrodynamite18 committed:
EOF newline removed.

Datagen was being finnicky on my PC, but I fixed it and the EOF newline is now properly gone.

nitrodynamite18 committed:
Merge branch 'patch-1' of https://github.com/nitrodynamite18/MinecraftForge into patch-1

TheCurle commented:
Sorry about the wait on this one.
